ui/app/src/components/ItemPublishingTargetIcon/ItemPublishingTargetIcon.tsx

```diff
@@ -44,14 +44,14 @@ const useStyles = makeStyles(() =>
       color: palette.gray.medium2,
       ...styles.root
     }),
-    publishingTargetLive: (styles) => ({
-      color: palette.green.main,
-      ...styles.publishingTargetLive
-    }),
```

Comment:
What's this for?

Reply:
When both live and staged are true (as we discussed yesterday), both styles apply to the PublishingTargetIcon component, and since the live styles were before the staging, it was showing the color for staging.

Reply:
We don't want both classes applied in those cases so instead of this, let's update the logic below to only include one. Let's roll this back.

Reply:
```js
item.stateMap.live ? classes.publishingTargetLive : item.stateMap.staged ? classes.publishingTargetStaged : false
```